lib/digitalocean/bootstrap/chart_values/nginx-ingress.j2.yaml

```diff
@@ -257,6 +257,7 @@ controller:
       service.beta.kubernetes.io/do-loadbalancer-algorithm: "least_connections"
       service.beta.kubernetes.io/do-loadbalancer-size-slug: "lb-small"
       service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"
+      service.beta.kubernetes.io/do-loadbalancer-hostname: lb.{{ managed_dns_domains_helm_format }}
```

Comment:
is the naming valid?

Reply:
that's what we have in external-dns chart